dotnet/src/Connectors/Connectors.Ollama.UnitTests/TestData/chat_completion_test_response.txt

```diff
@@ -0,0 +1 @@
+{"model":"phi3","created_at":"2024-07-02T12:30:00.295693434Z","message":{"role":"assistant","content":"This is test completion response"},"done_reason":"stop","done":true,"total_duration":69492224084,"load_duration":66637210792,"prompt_eval_count":10,"prompt_eval_duration":41164000,"eval_count":236,"eval_duration":2712234000}
```

Comment:
This is JSON so it should be a json file and well formatted so developers can read it

Reply:
This will be dropped, Ollama latest 3.0 dont use non-streaming apis.